ipmiscrape

```diff
@@ -18,6 +18,7 @@ POOL_SIZE = 30
 PROCESS_TIMEOUT = 1.5
 IPMI_USER_PASS_BY_ALIAS = {'*': ('ADMIN', 'ADMIN')}  # ~/.config/ipmikvm/dict
 IPMI_USER_PASS_BY_IP = {'*': ('ADMIN', 'ADMIN')}     # ~/.config/ipmikvm/dict
+CONF_PATH = f'{os.environ.get('XDG_CONFIG_HOME', '~/.config')}/ipmikvm'
```

Comment:
Unfixed problem with py3.10.

Also, here you're using CONF_PATH for a directory. Above we have DATA_DIR (for a directory). And when looking below we have CONF_PATH for a file (nbdig). Please choose more consistent naming.

Reply:
CONF_DIR is consistent with DATA_DIR, and conveys that it's a directory. So, yes, that will work for me. Then you can keep CONF_PATH in nbdig, or replace it with CONF_FILE.